members/templates/members/activity_signup.html

```diff
@@ -1,115 +1,115 @@
 {% extends "members/base.html" %}
+{% load static %}
 {% load crispy_forms_tags %}
 {% block content %}
     <div class="container-fluid">
-      <div class="row">
-        <div class="col-lg-12">
-          <h1>{{activity.department.name}} <small>{{activity.name}}</small></h1>
-          <p>
-            {{ activity.description|linebreaksbr }}
-          </p>
+      <h1>{{activity.name}}<br /><small>Coding Pirates {{activity.department.name}}</small></h1>      
+      
+      <br />
+
+      <div class="card">
+        <div class="card-body">
+          <h5 class="card-title">Aktiviteten</h5>
+          <p class="card-text">Aktiviteten afholdes af Coding Pirates {{activity.department.name}}</p>
+
+          <div class="row">
+            <span class="col-xs-12 col-sm-12 col-md-6"><i class="fa fa-calendar"></i> Periode: {{ activity.start_date|date:"j. F" }} - {{ activity.end_date|date:"j. F" }}</span>
+            <span class="col-xs-12 col-sm-12 col-md-6"><i class="fa fa-clock-o"></i> Tidspunkt: {{ activity.open_hours }}</span>
+            <span class="col-xs-12 col-sm-12 col-md-6"><i class="fa fa-money"></i> Pris: {{ price | floatformat:2 }}</span>
+            <span class="col-xs-12 col-sm-12 col-md-6"><i class="fa fa-users"></i> {{ seats_left }} af {{ activity.max_participants }} pladser ledige.</span>
+            <span class="col-xs-12 col-sm-12 col-md-6"><i class="fa fa-hourglass-end"></i> {{ activity.signup_closing }} (om {{ activity.signup_closing|timeuntil }})</span>
+            <span class="col-xs-12 col-sm-12 col-md-6"><i class="fa fa-list"></i> Venteliste:
+              {% if activity.open_invite %}
+                Åben invitation uden venteliste
+              {% else %}
+                Kræver invitation igennem ventelisten
+              {%endif%}</span>
+          </div>
+          <br />
+
+          <div class="row">
+            <div class="col-xs-12 col-sm-12 col-md-6">
+              <p class="card-text">
+                <b>Adresse</b><br />
+                  {{activity.streetname}} {{activity.housenumber}}{%if activity.floor or activity.door%}, {%endif%}
+                  {%if activity.floor %}{{activity.floor}}.{%endif%}
+                  {%if activity.door%}{{activity.door}}{%endif%}
+              </p>
+              <p class="card-text">Lokalforening: Coding Pirates {{ union.name }}</p>
+            </div>
+            
+
+            <div class="col-xs-12 col-sm-12 col-md-6">
+              <div class="card">
+                <div class="card-header">
+                  Kontakt aktivitetsansvarlig
+                </div>
+                <div class="card-body">
+                  <p>Har du spørgsmål vedrørende aktiviteten?</p>
+                  <p>
+                    <b>{{ activity.responsible_name }}</b> <br/>
+                    E-mail: {{ activity.responsible_contact }}
+                  </p>
+                </div>
```

Comment:
Skal vi overveje en lille tekst med noget i stil med "Bemærk personen er frivillig. Vi svarer hurtigst muligt, men der kan godt gå et par dage. Hvis du har brug for hjælp ikke direkte relateret til aktiviteten kan du kontakte Coding Pirates Denmark på kontakt@codingpirates.dk" - den sidste sætning kan vi overveje om skal med. Det kan også være intet af det skal med.

Reply:
Jeg tænker ikke det er nødvendigt at skrive, at svar kan tage et par dage.

Omrking kontakt til sekretariatet, så er idéen med kontaktkassen, at alt for mange e-mails omkring aktiviteten havner hos sekretariatet. Jeg har prøvet at gøre det tydeligt, at det kun er aktiviteten ved at skrive "Kontakt aktivitetsansvarlig"